internal/venice-test-common/src/main/java/com/linkedin/venice/utils/TestUtils.java

```diff
@@ -80,18 +80,9 @@
 import java.io.IOException;
 import java.net.ServerSocket;
 import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
 import java.security.Permission;
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
Can you revert the wildcard imports? You might need to update your intellij settings. (There's a bug in IntelliJ that these get reverted on each restart)